.github/workflows/continuous-integration-arm64.yml

```diff
@@ -1,48 +0,0 @@
-name: github-arm64
-
-on:
-  workflow_call:
-
-concurrency:
-  group: ${ {github.event_name }}-${{ github.workflow }}-${{ github.ref }}-arm64
-  cancel-in-progress: ${{github.event_name == 'pull_request'}}
-
-permissions: read-all
-
-jobs:
-  arm64ci:
-    name: arm64-ci
-    runs-on: [ubuntu-24.04-arm]
-    container:
-      image: ghcr.io/kokkos/ci-containers/ubuntu:latest-arm64
-
-    strategy:
-      matrix:
-        include:
-          - backend: "OPENMP"
-            cmake_build_type: "RelWithDebInfo"
-          - backend: "THREADS"
-            cmake_build_type: "RelWithDebInfo"
-          - backend: "SERIAL"
-            cmake_build_type: "Release"
-
-    steps:
-      - uses: actions/checkout@11bd71901bbe5b1630ceea73d27597364c9af683 # v4.2.2
-      - name: configure
-        run:
-          cmake -B build .
```

Comment:
@masterleinad, I just double check again:

- [ ] `-DKokkos_ENABLE_${{ matrix.backend }}=On`
- [ ] `-DCMAKE_CXX_CLANG_TIDY="clang-tidy;-warnings-as-errors=*"`
- [ ] `-DCMAKE_CXX_FLAGS="-Werror"`
- [ ] `-DCMAKE_CXX_STANDARD=17`
- [ ] `-DKokkos_ARCH_NATIVE=ON`
- [ ] `-DKokkos_ENABLE_COMPILER_WARNINGS=ON`
- [ ] `-DKokkos_ENABLE_DEPRECATED_CODE_4=OFF`
- [ ] `-DKokkos_ENABLE_TESTS=On`
- [ ] `-DCMAKE_BUILD_TYPE=${{ matrix.cmake_build_type }}`

I think we should have all the important option now!

Reply:
`Kokkos_ENABLE_DEPRECATED_CODE_4=OFF` should be fine.